internal/legacy/php_windows.go

```diff
@@ -19,57 +23,79 @@ var phpCLI []byte
 var caCert []byte
 
 // copyPHP to destination, if it does not exist
-func (c *CLIWrapper) copyPHP() error {
-	dest := path.Join(c.cacheDir(), "php")
-	br := bytes.NewReader(phpCLI)
-	r, err := zip.NewReader(br, int64(len(phpCLI)))
+func (c *CLIWrapper) copyPHP(cacheDir string) error {
```

Comment:
Why pass the `cacheDir` as a parameter here instead of getting it like before?

Reply:
Convenience, really, as the `cacheDir()` method now returns an error, and it creates the cache (sub)directory if necessary. These are unexported/internal functions. I thought passing the param helps to show that the cacheDir has to be set up before calling.

Reply:
I notice none of these `php_unix.go` and `php_windows.go` functions need to be struct methods - it might make it clearer if they were standalone/pure?

We could also go further and make something like a "phpManager" interface, which each OS-specific file has to implement.